WordPress/Classes/Extensions/NSAttributedString+Helpers.swift

```diff
@@ -1,16 +1,16 @@
 import Foundation
 
 
-extension NSAttributedString
-{
-    /**
-    *  @brief       Checks if a given Push Notification is a Push Authentication.
-    *  @details     This method will embed a collection of assets, in the specified NSRange's.
-    *               Since NSRange is an ObjC struct, you'll need to wrap it up into a NSValue instance!
-    *
-    *  @param       embeds  A colleciton of embeds. NSRange > UIImage.
-    *  @returns             An attributed string with all of the embeds specified, inlined.
-    */
+extension NSAttributedString {
+
+    /// Checks if a given Push Notification is a Push Authentication.
+    /// This method will embed a collection of assets, in the specified NSRange's.
```

Comment:
It took me a while to understand the description, starting at this line.

One suggestion to improve this: since this is actually documentation for the `embeds` parameter, why not add it there and specify you're referring to the key used for the `embeds` dictionary?